content/post/prioritization_meeting.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Prioritizing Prioritization"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["agile"]
+date: 2018-06-15T17:27:09-04:00
+draft: true
+---
+
+The sprint prioritization meeting is an integral aspect of agile philosophy. You may be more familiar with concepts like stand up, sprint planning, back log grooming and retros, but the prioritization meeting often garners less attention I suspect, partially because it is a difficult process to deploy. It involves regularly occurring meetings, thoughtful ticket (or story) descriptions, considerations of each ticket vis-a-vis every other ticket, and to top it off, this all needs to be coordinated among various Data stakeholders. There is a reason why scrum masters are typically referred to as [cat herders](https://www.google.com/search?ei=GVYpW66lCdHX5gLDsbegDg&q=%22scrum+master%22+%26+%22cat+herder%22&oq=%22scrum+master%22+%26+%22cat+herder%22&gs_l=psy-ab.3..33i160k1.5030.9792.0.10255.6.6.0.0.0.0.139.663.1j5.6.0....0...1c.1.64.psy-ab..0.1.139....0.M519ryGOesY).
```

Comment:
Here's my attempt at re-writing the whole graph

> The sprint prioritization meeting is integral to the agile process. While many people are familiar with the oft-heralded meetings like standup, sprint planning, backlog grooming, and retros, the sprint prioritization meeting often receives less attention. I suspect this is because sprint planning is a particularly challenging process to manage. A good sprint planning process requires thoughtful ticket descriptions written in advance, a collaborative review of each ticket in the context of all of the other tickets, and the buy-in and coordination among all of the analytics stakeholders.

> There is a reason why scrum masters are typically referred to as cat herders.

Reply:
You don't like `vis-a-vis`? :)  The re-write sounds good.